src/sentry/tasks/sentry_apps.py

```diff
@@ -321,5 +331,15 @@ def send_and_save_webhook_request(url, sentry_app, app_platform_event):
             error_id=resp.headers.get("Sentry-Hook-Error"),
             project_id=resp.headers.get("Sentry-Hook-Project"),
         )
+        if not sentry_app.is_published:
+            raise IgnorableSentryAppError("unpublished or internal app")
+
+        if resp.status_code == 503:
+            raise ApiHostError(six.binary_type(resp.status_code))
```

Comment:
@ceorourke Can use `ApiHostError.from_request` to raise the error? I want to be as consistent as possible with how we raise the errors for integrations.